provisioning/device/tests/Amqp/ProvisioningTransportHandlerAmqpTests.cs

```diff
@@ -0,0 +1,122 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Security.Authentication;
+using FluentAssertions;
+using Microsoft.Azure.Amqp.Framing;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.Devices.Provisioning.Client.UnitTests
+{
+    [TestClass]
+    [TestCategory("UnitTest")]
```

Comment:
Test category could be "unit" instead of "unitTest" for consistency across test files

Reply:
Good catch!